src/vivarium_gates_nutrition_optimization/data/extra_gbd.py

```diff
@@ -29,9 +29,24 @@ def load_lbwsg_exposure(location: str):
     data = data[data["year_id"] == 2019].drop(columns="year_id")
     return data
 
+@gbd.memory.cache
+def get_all_cause_yld_rate(location: str):
+    entity = utilities.get_entity("cause.all_causes.ylds")
+    location_id = utility_data.get_location_id(location)
+    data = vi_utils.get_draws(
+        "cause_id",
+        entity.gbd_id,
+        source=gbd_constants.SOURCES.COMO,
+        location_id=location_id,
+        decomp_step=gbd_constants.DECOMP_STEP.STEP_5,
+        gbd_round_id=gbd_constants.ROUND_IDS.GBD_2019,
+        measure_id=vi_globals.MEASURES["YLDs"],
+        metric_id=3 #rate
+    )
+    return data
 
 @gbd.memory.cache
-def get_maternal_disorder_ylds(location: str):
+def get_maternal_disorder_ylds(location: str, metric_id=None):
```

Comment:
why metric id None? And where in here aare you specifying cause 366?

Reply:
oh, that's entity.gbd_id

Reply:
The short answer is that that is the default value for metric_id in get_draws(), so that was what was being passed in previously. I'm just adding the ability to specifically pull YLD rate rather than total YLDs.

OTOH, it seems like GBD should be returning *every* metric_id when it is equal to None, which makes me wonder why it it works at all. It seems like by default the query for YLDs only has metric ID 3 available. I guess this isn't really strictly necessary, since it will get metric ID 3 regardless, I'm just reproducing what is in the documentation.